dxr/plugins/python/__init__.py

```diff
@@ -1,4 +1,16 @@
 """Python plugin, using the ast and tokenize standard library modules"""
 
-from .indexers import FileToIndex, mappings
-from .filters import TypeFilter, FunctionFilter
+from dxr.plugins import Plugin
+
+from .indexers import mappings, TreeToIndex
+from .filters import BasesFilter, DerivedFilter, FunctionFilter, TypeFilter
+
+
+plugin = Plugin(
+    filters=[BasesFilter, DerivedFilter, FunctionFilter, TypeFilter],
```

Comment:
Why not use `filters_from_namespace` here so you never have to update this again?